prombench/Makefile

```diff
@@ -27,6 +27,8 @@ cluster_resource_apply:
 		-v OAUTH_TOKEN="$(printf ${OAUTH_TOKEN} | base64 -w 0)" \
 		-v WH_SECRET="$(printf ${WH_SECRET} | base64 -w 0)" \
 		-v GITHUB_ORG:${GITHUB_ORG} -v GITHUB_REPO:${GITHUB_REPO} \
+		-v COLLECTOR_IMAGE_REPOSITORY:${COLLECTOR_IMAGE_REPOSITORY} \
```

Comment:
1. Can we have default value for this image defined in Makefile?
2. Let's call it PROMETHEUS_IMAGE_REPOSITORY` (: